style.css

```diff
@@ -0,0 +1,49 @@
+* {
+  margin: 0;
+  padding: 0;
+  background: #004882;
+}
+.box {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+.box select {
+  width: 250px;
+  height: 50px;
+  font-size: 20px;
+
+  padding: 10px;
+
+  background: #0563af;
+  color: #fff;
+
+  border: none;
+
+  box-shadow: 0 5px 25px rgba(0, 0, 0, 0.5);
+  appearance: none;
+  outline: none;
+}
+.box:before {
+  width: 50px;
+  height: 50px;
```

Comment:
Use relative units

Reply:
surely it is done please check @Abbas4tech